docs/BR.md

```diff
@@ -1326,21 +1326,32 @@ No stipulation.
 
 The following SHALL apply for communicating the status of Certificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
 
+Authoritative OCSP responses MUST be available (i.e. the responder MUST NOT respond with the "unknown" status) starting no more than 15 minutes after the Certificate's `notAfter` timestamp.
+
+The following SHALL apply for communicating the status of *all* Certificates for which an OCSP responder is willing to respond.
```

Comment:
What does "willing" mean? Could an OCSP response of always "unknown" be considered non-willing, and thus always be compliant?
```suggestion
The following SHALL apply for communicating the status of any certificate containing an Authority Information Access extension with an id-ad-ocsp accessMethod.
```

Reply:
Yes, that's the idea.

There are four possible situations:
- The cert contains an OCSP URL, and the responder provides responses. This is good.
- The cert contains an OCSP URL, but the responder does not provide responses (i.e. responds "unknown"). This is bad, and is covered by the clause above which says that responses MUST be available.
- The cert does not contain an OCSP URL, and the responder does not provide responses. This is fine, and is the whole point of Ballot SC-063 making CRLs mandatory and OCSP optional.
- The cert does not contain an OCSP URL, but the responder provides responses anyway. This _should be_ fine. But the current BRs have no provision for it. A responder which provides OCSP responses for certs which do not contain OCSP URLs is, per the BRs, allowed to make those OCSP responses wildly malformed. This is what this ballot is attempting to resolve, by stating that these requirements apply even to those responses.

Reply:
I've now completely rearranged this section, and I believe it should be much clearer now. Let me know what you think!